hack/flags.sh

```diff
@@ -59,6 +59,9 @@ parse_flags() {
 
   if [ "${MODE}" == "preview" ]
   then
+    if [ "${PARSE_FLAGS_RUN}" == "1" ]; then
+      return
```

Comment:
could you add a comment to explain why it returns - it doesn't have to be obvious for the first glance.